config/routes.rb

```diff
@@ -318,6 +318,12 @@
     resource :organisations, only: %i[new create]
   end
 
+  # for commissioner reporting app
+  post "/tokens/authorize", controller: :one_time_tokens, action: "authorize"
```

Comment:
Should this route also be under `/api`?

Reply:
yes, see the [above comment](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/3866#discussion_r2242128162) for details